peer_mentoring/account_management/admin.py

```diff
@@ -1,8 +1,23 @@
 from django.contrib import admin
-from .models import Interest, Photo, CustomUser, UserProfile
+from django.contrib.auth.admin import UserAdmin
+
+from .models import Interest, Photo, UserProfile
 
 # Register your models here.
-admin.site.register(Interest)
-admin.site.register(Photo)
-admin.site.register(CustomUser)
-admin.site.register(UserProfile)
\ No newline at end of file
+@admin.register(Interest)
+class InterestAdmin(admin.ModelAdmin):
+    list_display = ('id', 'name', 'image')
+
+
+@admin.register(Photo)
+class PhotoAdmin(admin.ModelAdmin):
+    list_display = ('id', 'image', 'description', 'user')
+
+
+@admin.register(UserProfile)
+class UserProfileAdmin(admin.ModelAdmin):
+    list_display = ('id', 'user', 'avatar', 'title', 'bio')
+
+
```

Comment:
adding pre-commit will get rid of these spaces and will add 2 lines between classes consistently as per PEP8 (btw this is brilliant: https://twitter.com/bbelderbos/status/1634166553473064960)